### ISSUE TITLE
[Enhancement] Support for ProRPGItems Damages

### ISSUE THREAD

**Issue by lordsrubka5** [magemonkeystudio/fabled#217]
**Describe the solution you'd like**

A way for SkillAPI attributes to increase the damage of prorpgitem damage types.

**Comment by goflishMC** [comment]
I don't think we would start a project like this until the damage system has been rewritten in prorpgitems.

**Comment by Travja** [comment]
Adding a stat that modifies `rpgdamage-<type>` or `rpgdefense-<type>` can do this
